common/types/metrics.ts

```diff
@@ -5,6 +5,12 @@
 
 import { VisualizationType } from './custom_panels';
 
+// export interface MetricData {
+//   metricId: string;
+//   metricType: 'savedCustomMetric' | 'prometheusMetric' | 'openTelemetryMetric';
+//   metricName: string;
+// }
+
```

Comment:
Can we please remove the commented code lines?

Reply:
removed